tests/test_main.py

```diff
@@ -1 +1,84 @@
-# write your code here
+import datetime
+import random
+
+import pytest
+
+from app.main import actual_password
+
+
+def thirty_days_or_less():
+    today = datetime.date.today()
+    random_day = random.randint(1, 30)
```

Comment:
Do not use `random` in tests without a specific reason. Tests become flaky  :)

Reply:
I've fixed